src/components/TodoList/TodoList.tsx

```diff
@@ -1,12 +1,27 @@
 /* eslint-disable max-len */
 import React from 'react';
+import { useAppSelector } from '../../app/hooks';
+import { Todo } from '../Todo/todo';
+import { filterTodos } from '../filteredTodos/filteredTodos';
+
+type Props = {
+  // wygodniej dac any bo nazwa typu Todo pokrywa sie z nazwa komponentu Todo
+  todos: any[];
```

Comment:
again, if you're typing everything as any, using TS has basically no point, I know it's convenient in some particular cases, but most of the time there are many workarounds